ur_moveit_config/config/ur10/joint_limits.yaml

```diff
@@ -0,0 +1,34 @@
+# joint_limits.yaml allows the dynamics properties specified in the URDF to be overwritten or augmented as needed
+# Specific joint properties can be changed with the keys [max_position, min_position, max_velocity, max_acceleration]
+# Joint limits can be turned off with [has_velocity_limits, has_acceleration_limits]
```

Comment:
I think we can add acceleration, effort, and jerk limits in a followup PR, eventually  :+1: